Sources/Impl/PinLayoutImpl.swift

```diff
@@ -234,20 +234,63 @@ class PinLayoutImpl: PinLayout {
         left({ "all() left coordinate" })
         return self
     }
+
+    @discardableResult
+    func all(_ value: CGFloat) -> PinLayout {
+        func context() -> String { return "all(\(value))" }
+        top(value, context)
```

Comment:
Could you do as in `all()`, i.e. set a different context string for all internal calls (ex: { "all(\(value)) top coordinate" })